udf/agent/udf.pb.go

```diff
@@ -1,50 +1,29 @@
-// Code generated by protoc-gen-go.
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
Did we bump the protoc-gen-go version and trigger this change? Or is this a gofmt thing?

Reply:
I had to mess with the versions a little bit due to some broken things.